pkg/controller/nodes/common/utils.go

```diff
@@ -0,0 +1,36 @@
+package common
```

Comment:
unit tests?

Reply:
Sure. It is covered by few tests. Will add one for this as well.